scripts/test/runtests.ps1

```diff
@@ -44,7 +49,7 @@ pushd "$TestBinRoot"
 
 # Run each test project
 $TestProjects | ForEach-Object {
-    & "corerun.exe"  "xunit.console.netcore.exe" "$_.dll" -xml "$_.xml" -notrait category=failing
+    & "corerun.exe"  "xunit.console.netcore.exe" "$_.dll" -xml "$_.xml" -notrait category=failing -parallel none
```

Comment:
Agreed. Let's leave parallel on. If someone writes a buggy test they will fix it.